regression/run/references/main.f

```diff
@@ -0,0 +1,46 @@
+! Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+! other Shroud Project Developers.
+! See the top-level COPYRIGHT file for details.
+!
+! SPDX-License-Identifier: (BSD-3-Clause)
+! #######################################################################
+!
+! Test code generated by the references test.
```

Comment:
@ltaylor16 -- is this file auto-generated? Or just stubs for it?

Reply:
This file is not auto-generated.  Parts of it are boilerplate for fruit. It tests the generated Fortran API.

Reply:
Thanks for clarifying @ltaylor16, that's what I suspected.

The line that I commented on is a bit confusing.
I think the intent is that it 
> _tests_ the shroud-generated code for the 'references' test,

but it can be read as:
> _test code_ [that is] generated by the 'references' test

(I'm not trying to be pendantic -- the comment confused me a bit).

Reply:
It's always good to get another perspective because it made perfect sense to me when I wrote it.  I find I often update comments six months after I've written them so they are less ambiguous.